debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java

```diff
@@ -402,13 +403,23 @@ public void shouldReceiveChangesForDefaultValues() throws Exception {
         String statements = "ALTER TABLE test_table REPLICA IDENTITY FULL;" +
                             "ALTER TABLE test_table ADD COLUMN default_column TEXT DEFAULT 'default';" +
                             "INSERT INTO test_table (text) VALUES ('update');";
+        Thread.sleep(50_000);
         consumer = testConsumer(1);
         recordsProducer.start(consumer, blackHole);
         executeAndWait(statements);
 
         SourceRecord insertRecord = consumer.remove();
+        System.out.println(insertRecord);
```

Comment:
Can we get rid of the println calls here and below? Let's rather add some actual assertion. Or is this some sort of left-over?